src/data/localization.rs

```diff
@@ -285,6 +285,41 @@ impl Localization {
         }
     }
 
+    #[cfg(feature = "ck3")]
+    pub fn verify_name_exists(&self, name: &Token, max_sev: Severity) {
+        if name.as_str().is_empty() {
+            return;
+        }
+        self.mark_used(name.as_str());
+        let mut langs: Vec<&str> = Vec::new();
+        for lang in &self.mod_langs {
+            let hash = self.locas.get(lang);
+            if hash.is_none() || !hash.unwrap().contains_key(name.as_str()) {
+                langs.push(lang);
+            }
+        }
+        if !langs.is_empty() {
+            // It's merely untidy if the name is only missing in latin-script languages and the
+            // name doesn't have indicators that it really needs to be localized (such as underscores
+            // or extra uppercase letters). In all other cases it's a warning.
+            let sev = if only_latin_script(&langs)
```

Comment:
This is to assume that name only contains latin chars, which may not be the case. For example, if the name is cyrillic and the mod only supports Russian, then this will be warning still, even though it doesn't need to be. On the other hand, if name is Chinese characters, and missing Latin locas, this would only be untidy. Simply checking ascii doesn't work either, since there could be accented characters.

An improvement could be to use unicode blocks to guess the language, e.g. [unicode_blocks](https://docs.rs/unicode-blocks/latest/unicode_blocks/).

Reply:
Hmm yeah. It's built around the assumption that the input name is English